dawn/src/dawn4py/__init__.py

```diff
@@ -22,7 +22,7 @@
 
 from . import _dawn4py
 from . import serialization
-from ._dawn4py import Options, SerializerFormat
+from ._dawn4py import *
```

Comment:
why `from X import *`
https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import/710603#710603

Reply:
Switched to naming these individually